### PR TITLE
Remove typescript tutorial

### PR DESCRIPTION
**Pull request by TheZoker**
- Update tutorials (#83)
- Remove typescript tutorial
- Modify basic and redux tutorial
- Add redirect from ` /docs/tutorial-typescript` to `/docs/tutorial-redux` (**needs testing**)
- Add subpage to docs TOC
- Fix console errors

### CODE REVIEW

File: src/components/DocsToc.js [hunk 1]
```diff
@@ -22,17 +22,50 @@ const additionalStyles = {
...
+      <li key={menu.id} style={style}>
+        <MenuLink to={menu.route} label={menu.name} />
+      </li>
+    );
+  }
+}
+
 const DocsToc = ({ classes }) => {
-  const toc = groupByParent(useMenus({ filter: m => m.route.startsWith('/docs/')}))
+  const [query] = useState('');
```
Comment (sdirix) [inline src/components/DocsToc.js]: I know this is copied but it just seems useless to me. Can we try whether the current behavior is preserved without this state variable and evaluating the ternary below always to false?

If it still works please also fix it where this is coming from.

File: src/components/DocsToc.js [hunk 2]
```diff
@@ -1,8 +1,8 @@
-import React from 'react';
+import React, { useState } from "react";
```
Comment (sdirix) [inline src/components/DocsToc.js]: somehow this has `"` instead of `'`

**Review by sdirix — CHANGES_REQUESTED** [review]
Works in general. Thanks!

I synced the dev and master branch as they accidentally diverged, however this means you need to cherry-pick Alex's commit and your commits back on the current development branch.

I tested the redirection and it works. Nice!

The "Fix console errors" commit also contains changes unrelated to that, so please split the commit up.

Please check the remaining comments.

File: src/components/DocsToc.js (hunk 1 above)
Comment (TheZoker) [reply src/components/DocsToc.js]: I removed query

File: src/components/DocsToc.js (hunk 2 above)
Comment (TheZoker) [reply src/components/DocsToc.js]: Done

**Comment by TheZoker** [comment]
Done, all comments are implemented

**Review by sdirix — APPROVED** [review]
Works great! Thanks!

**Pull request merged**